app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/RedirectUrlActions.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscriptionfrontend.controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.Logger
+import play.api.mvc._
+import uk.gov.hmrc.agentsubscriptionfrontend.connectors.SsoConnector
+import uk.gov.hmrc.agentsubscriptionfrontend.service.SessionStoreService
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.binders.RedirectUrl._
+import uk.gov.hmrc.play.bootstrap.binders.{AbsoluteWithHostnameFromWhitelist, RedirectUrl, UnsafePermitAll}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.higherKinds
+import scala.util.{Failure, Success, Try}
+
```

Comment:
Hey, I know comments are seen as an anti-pattern, but it would be really nice to have a simple comment block at the top of this file which describes exactly what a "RedirectUrl" is in this context.  People might think it's just a standard HTTP 303, but I think the intention here is something different?  This is regarding what happens at the end of a journey, right?  Since RedirectUrl is such a general term, I think a comment would be helpful :)

Reply:
👍